### ISSUE TITLE
Think how to log exceptions...

### ISSUE THREAD

**Issue by LenarBad** [LenarBad/easylog#48]

**Comment by LenarBad** [comment]
The simplest way is just ```catch, log stack trace, throw```

```java
        Object result = null;
        try{
            result = jp.proceed(jp.getArgs());
        }catch(Exception e){
            logger.error(e);
            throw(e);
        }
```
https://github.com/LenarBad/easylog/blob/162e8d4846a5fa548847b6aeec78bbc0b4c98bd4/src/main/java/io/lenar/easy/log/UneasyLogger.java#L35

**Comment by LenarBad** [comment]
PR:  [Exceptions logging - https://github.com/LenarBad/easylog/pull/50](https://github.com/LenarBad/easylog/pull/50)

**Comment by LenarBad** [comment]
Released 1.1.5
